style.css

```diff
@@ -15,6 +23,48 @@ body {
   white-space: nowrap;
 }
 
+.menu-popup {

```

Comment:
- [ ] This is a nav menu popup window, so it should be fixed in the viewport. Otherwise, the user has to scroll to this menu.
- [ ] You have hard-coded many attributes which are resulting in inconsistent responsiveness,
- You can update this entire CSS to the following:
```
    background-color: #6070ff;
    position: fixed;
    top: 0;
    bottom: 0;
    right: 0;
    left: 0;
    display: flex;
    flex-direction: column;
    backdrop-filter: blur(6px);
    align-items: start;
    font-weight: 600px;
    font-size: 32px;
    line-height: 44px;
```

Reply:
@jasonfelice hi, 
Thank you so much for your review!
As per your review I have made changes to my code. Could you please re-review?